packages/mux-uploader/src/mux-uploader.ts

```diff
@@ -95,8 +86,7 @@ template.innerHTML = `
   ${styles}
 </style>
 
-<div class="sr-only" id="sr-only" aria-live="polite"></div>
-
+<mux-uploader-sr-text></mux-uploader-sr-text>
 <div class=text-container>
```

Comment:
idea(non-blocking): It's feeling like we should have some kind of state attribute+property for at least the upload state. Off the dome finite state possibilities: PENDING, PROGRESS (or UPLOADING etc.), SUCCEEDED (or UPLOADED etc.), FAILED (or ERROR etc.).

Reply:
this kind of ties into the nested shadow doms issue where if we wanted a .sr-only utility class we don't have an easy way of sharing it. Both since we can't include it at the top level of an element and expect it to be available to nested shadow doms, but also, we don't currently have a way to do CSS includes (though, I believe this is solvable via esbuild). Though, that could mean we're including the same utility multiple times.

Reply:
Totally, this is scoped and will be in a future revision